R/bundleFiles.R

```diff
@@ -127,14 +87,25 @@ bundleFiles <- function(appDir) {
   listBundleFiles(appDir)$contents
 }
 
+explodeFiles <- function(dir, files) {
+  files <- files[file.exists(file.path(dir, files))]
```

Comment:
This makes it very clear that `explodeFiles()` is doing something weird. Should I make this a warning or error?

Reply:
A warning is likely to be better, since this was previously silent.